modules/io/common/src/test/java/org/locationtech/jts/io/geojson/GeoJsonTest.java

```diff
@@ -92,11 +94,55 @@ public void testGeometryCollectionEmpty() throws ParseException {
     runTest("GEOMETRYCOLLECTION EMPTY");
   }
 
+  public void testFeature() throws ParseException {
```

Comment:
Shouldn't there be some tests which have actual property data, to show they can be handled?